runtime/vm/LayoutFFITypeHelpers.hpp

```diff
@@ -75,7 +79,49 @@ class LayoutFFITypeHelpers
 	}
 
 	/**
-	 * @brief Convert argument or return type from the type of ffi_type to J9NativeTypeCode
+	 * @brief Get the length of the specified struct layout string by counting the bracket symbols
+	 * for the struct to compute the length.
+	 *
+	 * For example, if a struct layout string is "#2[II]", it returns 6.
+	 *
+	 * @param layout[in] A pointer to a C string describing the types of the struct elements,
+	 * which points to the end of the struct layout string upon return
```

Comment:
Please remove the clause "which points to..." which isn't meaningful to callers as `layout` is not an "out" parameter.

Reply:
Fixed.